model/src/main/proto/interaction_object.proto

```diff
@@ -18,6 +18,7 @@ message InteractionObject {
     // wrapped into a new message.
     StringList set_of_html_string = 7;
     Fraction fraction = 8;
+    ListOfSetsOfHtmlStrings list_of_sets_of_html_string = 9;
```

Comment:
Can we just replace this with repeated StringList directly or do we want to keep it a separate interaction object type itself for other reasons? @BenHenning

Reply:
We can't directly use the repeated keyword it was written as a comment

Reply:
Ah okay that makes sense, thanks!